src/gsi/crtm_interface.f90

```diff
@@ -2597,6 +2667,186 @@ subroutine calc_gfdl_reff(rho_air,tsen,qxmr,cloud_name,reff)
 
   end subroutine calc_gfdl_reff
 
+  subroutine calc_thompson_reff(rho_air,tsen,qxmr,cloud_name,reff)
+!$$$  subprogram documentation block
+!                .      .    .                                       .
+! subprogram:    calc_thompson_reff:  calculate effective radius based on Thompson scheme
+!   prgmmr:      Azadeh.Gholoubi
+!
+! program history log:
+!
+!   2023-01-24
+!
+!   language: f90
+!
+!$$$
+!--------
+  use constants, only: zero, pi
+  implicit none
+
+! Declare passed variables
+  character(10)                 ,intent(in   ) :: cloud_name
+  real(r_kind), dimension(nsig) ,intent(in   ) :: rho_air   ! [ kg/m3  ]
+  real(r_kind), dimension(nsig) ,intent(in   ) :: tsen      ! [ K      ]
+  real(r_kind), dimension(nsig) ,intent(in   ) :: qxmr      ! [ kg/kg  ]
+  real(r_kind), dimension(nsig) ,intent(inout) :: reff      ! [ micron ]
+
+! Declare local variables
+  character(len=*), parameter :: myname_ = 'calc_thompson_reff'
+  integer(i_kind) :: k
+  integer(i_kind) :: j
```

Comment:
Compilation in debug mode flagged `j` as not being used in this routine
```
/lfs/h2/emc/da/noscrub/russ.treadon/git/gsi/pr743/src/gsi/crtm_interface.f90(2697): remark #7712: This variable has not been used.   [J]
  integer(i_kind) :: j
```
Please remove this line.  

NCO builds code in debug mode.  If their compilation finds unused variables, the code will be sent back to the developer(s).